ComponentKit/Utilities/CKInternalHelpers.mm

```diff
@@ -35,7 +35,7 @@ NSUInteger CKIntegerArrayHash(const NSUInteger *subhashes, NSUInteger count)
 {
   NSUInteger result = subhashes[0];
   for (int ii = 1; ii < count; ++ii) {
-    result = std::hash<unsigned long long>()((((unsigned long long)result) << 32 | subhashes[ii]));
+    result = std::hash<unsigned long long>()((((unsigned long long)result) << sizeof(NSUInteger) | subhashes[ii]));
```

Comment:
Yeah, I think you are right about it. It’s hard to say which one is better. In results, they can get same output here. The reason I chose sizeof(NSUInteger) just because of reminding me each element is NSUInteger. However, I have to honestly admit that sizeof(unsigned long long) looks much better when having the first sight of it :)